src/Config/ConfigReader.cpp

```diff
@@ -153,13 +153,18 @@ WalletConfig ConfigReader::ReadWalletConfig(const Json::Value& root, const EEnvi
 	const std::string walletPath = dataPath + "WALLET/";
 
 	std::string databaseType = "ROCKSDB";
+	uint32_t minimumConfirmations = 10;
 	if (root.isMember(ConfigProps::Wallet::WALLET))
 	{
 		const Json::Value& walletRoot = root[ConfigProps::Wallet::WALLET];
 		if (walletRoot.isMember(ConfigProps::Wallet::DATABASE))
 		{
 			databaseType = walletRoot.get(ConfigProps::Wallet::DATABASE, "ROCKSDB").asString();
 		}
+		if (walletRoot.isMember(ConfigProps::Wallet::MIN_CONFIRMATIONS))
+		{
+			minimumConfirmations = walletRoot.get(ConfigProps::Wallet::MIN_CONFIRMATIONS, 10).asUInt();
```

Comment:
So this code isn't actually doing anything other than reading the value and ignoring it. It's up to you if you want to code the rest (needs added to WalletConfig, needs to be read in the Wallet submodule when calculating status, etc, or if you just want me to (I actually already had all the code locally - I just don't keep github up-to-date as well as I should)